docs/DeveloperGuide.md

```diff
@@ -8,22 +8,107 @@
 
 ### Value proposition
 
-{Describe the value proposition: what problem does it solve?}
+Grocery in Time aims to act as an easy-to-use central database for all the user's groceries. Managing many groceries stored at different locations around the house can get confusing,
+therefore our app will allow users to track their groceries easily. 
+
+Users are able to edit and manage the category, amount, expiration date, and storage location of their groceries.
+When groceries are running low, the app can generate a shopping list to remind users of what they need to buy.
+Furthermore, the app can generate a list of items that are expiring soon, reminding users to consume their groceries as soon as possible.
 
 ## User Stories
 
-|Version| As a ... | I want to ... | So that I can ...|
-|--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+| Version | As a ...                      | I want to ...                               | So that I can ...                                      |
+|---------|-------------------------------|---------------------------------------------|--------------------------------------------------------|
+| v1.0    | new user                      | see instructions on how to use the app      | refer to them when I forget how to use the application |
+| v1.0    | user                          | add groceries to the app                    | manage all my groceries                                |
+| v1.0    | user                          | view all my groceries                       | know what I have bought                                |
+| v1.0    | user                          | delete groceries from the list              | stop tracking those groceries                          |
+| v1.0    | user                          | add the amount of a grocery                 | keep track of the amount of that item I have           |
+| v1.0    | user                          | add the expiration date of the grocery      | keep track of when my items expire easily              |
+| v2.0    | financially-aware user        | add the cost of the groceries               | know how much I am spending                            |
+| v2.0    | health-conscious user         | categorise my groceries                     | know what types of groceries I have                    |
+| v2.0    | user with many storage spaces | add the location of where an item is stored | see where I keep my groceries                          |
+| v2.0    | user who consumes groceries   | track the usage of my groceries             | know how much I have left                              |
+| v2.0    | user who cooks with recipes   | create and keep my own version of recipes   | refer to my own recipes when I cook                    |
```

Comment:
After going through the user stories, I am not sure how the calories management sequence diagram fits into any of the user stories. Was this missed out or the calories management is a new feature?